portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,13 +37,28 @@
   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> comments = new ArrayList<>();
-    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
-    PreparedQuery results = this.datastore.prepare(query);
-    for (Entity entity : results.asIterable()) {
-      comments.add((String) entity.getProperty("content"));
+    int limit = 3;
+    String ERR_MSG = "Comment limit must be a non-negative integer.";
```

Comment:
consider making this a static class member. Since it doesn't change, we don't need a new copy each time this method is called.

Reply:
Done!